entity/contents/filter_contents.h

```diff
@@ -91,4 +106,37 @@ class BlendFilterContents : public FilterContents {
   FML_DISALLOW_COPY_AND_ASSIGN(BlendFilterContents);
 };
 
+/*******************************************************************************
+ ******* GaussianBlur1DFilterContents
+ ******************************************************************************/
+
+class GaussianBlur1DFilterContents final : public FilterContents {
+ public:
+  GaussianBlur1DFilterContents();
+
+  ~GaussianBlur1DFilterContents() override;
+
+  void SetRadius(Scalar radius);
+
+  void SetDirection(Point direction);
+
+  void SetExpandBorder(bool expand);
+
+ private:
+  // |FilterContents|
+  bool RenderFilter(const std::vector<std::shared_ptr<Texture>>& input_textures,
+                    const ContentContext& renderer,
+                    RenderPass& pass) const override;
+
+  // |FilterContents|
+  virtual ISize GetOutputSize(
+      const InputTextures& input_textures) const override;
+
+  Scalar radius_;
```

Comment:
C++ Nit: Default all scalars in the header please (including the `expand_` below).

Reply:
Done.